index.html

```diff
@@ -2037,33 +2051,38 @@ <h3>Verify Proof</h3>
       <section>
         <h3>Verify Proof Sets and Chains</h3>
         <p>
-In a proof set or proof chain, a <a>secured data document</a> has a
-<code>proof</code> attribute which contains a list of proofs
+In the case of a proof set or proof chain a <a>secured data document</a> has a 
+<code>proof</code> attribute which contains a list of proofs 
 (<var>proof_list</var>).
 The following algorithm specifies how to check the authenticity and
 integrity of a <a>secured data document</a> by verifying each
 proof in the <var>proof_list</var>. Required inputs are a
-<a>secured data document</a> (<var>securedDocument</var>). A list of
+<a>secured data document</a> (<var>securedDocument</var>). A list of 
 verification results is produced as output.
         </p>
         <ol class="algorithm">
           <li>
-Set <var>proof_list</var> to </var><var>securedDocument</var>.<var>proof</var>.
+Set <var>proof_list</var> to </var><var>securedDocument</var>.<var>proof</var>. 
           </li>
           <li>
-For each <var>proof</var> in <var>proof_list</var>, run steps 2&ndash;11 of the algorithm in
-section <a href="verify-proof"></a>, and if no exceptions are raised, associate
+For each <var>proof</var> in <var>proof_list</var> run steps 2-11 of the algorithm in
+section <a href="verify-proof"></a> and if no exceptions are raised, associate
```

Comment:
The change above reverts a prior suggestion from me. I'm making a slightly different suggestion here.
```suggestion
For each <var>proof</var> in <var>proof_list</var>, run steps 2 through 11 of the algorithm in
section <a href="verify-proof"></a>; if no exceptions are raised, associate
```